.github/workflows/build.yml

```diff
@@ -0,0 +1,129 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: Build
+
+on:
+  push:
+    branches: [ "2.4.x" ]
+  pull_request:
+    branches: [ "2.4.x" ]
+
+jobs:
+  build:
+    env:
+      GIT_REF: ${{ github.ref }}
+      GIT_SHA: ${{ github.sha }}
+      Configuration: Release
+      SolutionFile: .\src\NodaTime-Web.sln
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Check out our repo
+      uses: actions/checkout@v3
+
+    - name: Install .NET 6
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '6.0.x'
+        include-prerelease: false
+        
+        
+    - name: Restore dependencies
+      run: dotnet restore $Env:SolutionFile
+    
+    - name: Build
+      run: dotnet build --no-restore  $Env:SolutionFile -c $Env:Configuration
+    
+    - name: Test
+      run: dotnet test --no-build --verbosity normal $Env:SolutionFile -c $Env:Configuration
+    
+    - name: Pack
+      run: dotnet pack $Env:SolutionFile --no-restore --no-build --configuration $Env:Configuration  -o ./nuget
+
+    - name: Sign packages
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
+      env:
+        TIMESTAMPER_URL: ${{ secrets.CODE_SIGN_CERTIFICATE_TIMESTAMPER_URL }}
+        PFX_BASE64: ${{ secrets.CODE_SIGN_CERTIFICATE_BASE64 }}
+        PFX_PASS: ${{ secrets.CODE_SIGN_CERTIFICATE_PASSWORD }}
+      run: |
+        $codesign_pfx = "code_sign_cert.pfx"
+        $bytes = [Convert]::FromBase64String($Env:PFX_BASE64)
+        [IO.File]::WriteAllBytes($codesign_pfx, $bytes)
+
+        Get-ChildItem ".\nuget\*.nupkg" -Recurse | ForEach-Object {
+          dotnet nuget sign $_.FullName --certificate-path $codesign_pfx --certificate-password $Env:PFX_PASS --timestamper $Env:TIMESTAMPER_URL
+        }
+        
+    - name: Configure Azure Artifacts feed
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
```

Comment:
The `buildVariables` step is missing.

If the plan is to push a new version of the package each time a commit is made to the branch, this condition can be simplify to just check if the workflow is executing in a pull request context or not.

Reply:
Ok. Condition changed.